community_tutorials_and_guides/blazingsql/bsql_vs_pyspark_netflow.ipynb

```diff
@@ -84,20 +84,20 @@
     "import os\n",
     "\n",
     "# relative path to data folder\n",
-    "data_dir = '../../../data/blazingsql/'\n",
+    "data_dir = './blazingsql/'\n",
```

Comment:
Curious about why you changed the path?

Reply:
Needed to update file paths to correctly point to where the data is stored.